spec/lib/jobs/clear-sel-by-usage-spec.js

```diff
@@ -0,0 +1,128 @@
+        this.sandbox.stub(ipmitool, 'selInformation').resolves(mockData);
+        this.sandbox.spy(_job, '_done');
+        return _job._run()
+        .then(function(){
+            expect(waterline.obms.findByNode).to.be.calledOnce;
+            expect(waterline.obms.findByNode).to.be.calledWith(
+                context.target, 'ipmi-obm-service', true
+            );
+            expect(ipmitool.selInformation).to.be.calledOnce;
+            expect(ipmitool.selInformation).to.be.calledWith(
+                obm.config.host,obm.config.user,obm.config.password
+            );
+            expect(parser.parseSelInformationData).to.be.calledOnce;
+            expect(parser.parseSelInformationData).to.be.calledWith(mockData);
+            expect(ipmitool.clearSel).not.to.be.called;
+            expect(_job._done).to.be.calledOnce;
+        });
+    });
+
+    it("should run job by clearing sel", function() {
+        var _options = _.cloneDeep(options);
+        _options.maxSelUsage = 50;
+        var _job = new ClearSelJob(_options, context, taskId);
+        this.sandbox.stub(ipmitool, 'selInformation').resolves(mockData);
+        return _job._run()
+        .then(function(){
+            expect(waterline.obms.findByNode).to.be.calledOnce;
+            expect(waterline.obms.findByNode).to.be.calledWith(
+                context.target, 'ipmi-obm-service', true
+            );
+            expect(ipmitool.selInformation).to.be.calledOnce;
+            expect(ipmitool.selInformation).to.be.calledWith(
+                obm.config.host,obm.config.user,obm.config.password
+            );
+            expect(parser.parseSelInformationData).to.be.calledOnce;
+            expect(parser.parseSelInformationData).to.be.calledWith(mockData);
+            expect(ipmitool.clearSel).to.be.calledOnce;
+            expect(ipmitool.clearSel).to.be.calledWith(
+                obm.config.host,obm.config.user,obm.config.password
+            );
+        });
+    });
+
+    it("should throw can not get node sel usage status error", function() {
+        var _mockData = "sel \n Percent Used 25% : Anything";
+        this.sandbox.stub(ipmitool, 'selInformation').resolves(_mockData);
+        var _job = new ClearSelJob(options, context, taskId);
+        this.sandbox.stub(_job, '_done');
+        return _job._run()
+        .then(function(){
+            expect(_job._done).to.callOnce;
+            expect(_job._done.args[0][0].message).to.equal("Can not get node SEL usage status");
+        });
+    });
+
+    it("should throw sel is almost full error", function() {
+        this.sandbox.stub(ipmitool, 'selInformation').resolves(mockData);
+        var _job = new ClearSelJob({maxSelUsage: 50}, context, taskId);
+        this.sandbox.stub(_job, '_done');
+        return _job._run()
+        .then(function(){
+            var err = new Error("Sel usage is 57\%, exceeds max sel usage 50\%");
+            expect(_job._done).to.callOnce;
```

Comment:
As I remember, there is no `callOnce` in sinon, the correct usage is "to.be.calledOnce". 

The chain function style (xxx.xxx.xxx) cannot detect the typo error, so unit test will not catch this error.